sdk/core/System.Memory.Data/src/System.Memory.Data.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <Description>Contains the BinaryData type, which is useful for converting between strings, streams, JSON, and bytes.</Description>
     <AssemblyTitle>System.Memory.Data</AssemblyTitle>
-    <Version>1.0.0</Version>
+    <Version>1.0.1-beta.1</Version>
```

Comment:
Do we need a beta.1 here? We usually just release the patch version.

Reply:
Yes, it was suggested that I add this just as an extra safeguard but as long as the release date is not set, there is no danger of it being released by accident. Will remove.